mkt/developers/views.py

```diff
@@ -1153,3 +1174,101 @@ def verify_data(self, data):
             return False
 
         return True
+
+
+class ContentRatingsPingbackV2(CORSMixin, SlugOrIdMixin, CreateAPIView):
```

Comment:
Since you're implementing your own `post` method I don't think you need `CreateAPIView`, just `APIView` or maybe `GenericAPIView`